lib/ReactViews/Map/Panels/HelpPanel/HelpPanelItem.jsx

```diff
@@ -49,19 +49,18 @@ class HelpPanelItem extends React.Component {
       flex-shrink: 0;
       width: 64px;
       height: 64px;
-      margin-right: 10px;
       display: table-cell;
+      vertical-align: middle;
+      padding-left: 25px;
     `;
     const CompassIcon = styled(Icon)`
-      position: absolute;
-      top: 50%;
-      left: 50%;
-      transform: translate(-50%, -50%);
+      position: relative;
+      transform: translate(0%, 0%);
       ${props =>
         `
-          fill: #575757;
-          width: 30px;
-          height: 30px;
+          fill: #4B4A4A;
+          width: 27px;
```

Comment:
Similar to StyledIcon.jsx mention above

Reply:
(You can still do this extra stuff like you have, it would just be something like.. `styled(StyledIcon)`)